### PR TITLE
lite: add tensorflowlite_flex to minimal example

### PR DESCRIPTION
**Pull request by aflaischer**
add option to link tensorflowlite_flex when building minimal example

### CODE REVIEW

**Comment by gbaned** [comment]
Hi @LukeBoyer Can you please review this PR ? Thank you!

File: tensorflow/lite/examples/minimal/CMakeLists.txt [hunk 1]
```diff
@@ -39,6 +41,20 @@ set(CMAKE_CXX_STANDARD 17)
...
+
+if(LINK_TFLITE_FLEX)
+  find_library(TF_LIB_FLEX tensorflowlite_flex HINTS "${TENSORFLOW_SOURCE_DIR}/bazel-bin/tensorflow/lite/delegates/flex/")
+  if(NOT TF_LIB_FLEX)
+    message(FATAL_ERROR "tensorflowlite_flex library not found")
+  endif()
+
+  target_link_libraries(minimal
+    tensorflow-lite
+    -Wl,--no-as-needed # Add --no-as-needed since for some toolchains (e.g Ubuntu) --as-needed is added by default
```
Comment (LukeBoyer) [inline tensorflow/lite/examples/minimal/CMakeLists.txt:53]: add period to comment

**Review by LukeBoyer — APPROVED** [review]

**Comment by gbaned** [comment]
Hi @aflaischer Can you please check @LukeBoyer's comments and keep us posted? Thank you!

File: tensorflow/lite/examples/minimal/CMakeLists.txt (hunk 1 above)
Comment (aflaischer) [reply tensorflow/lite/examples/minimal/CMakeLists.txt]: ok done

**Comment by aflaischer** [comment]
Hello @gbaned, I've updated the review

**Comment by gbaned** [comment]
Hi @LukeBoyer Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @LukeBoyer Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @LukeBoyer Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @LukeBoyer Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @LukeBoyer Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @LukeBoyer Can you please review this PR ? Thank you!

**Comment by aflaischer** [comment]
hi @gbaned, is there something I can do regarding the pull request?

**Comment by gbaned** [comment]
> hi @gbaned, is there something I can do regarding the pull request?

Hi @aflaischer Sorry for the delay, this PR is awaiting review status and will be processed further once it is approved. Nothing is pending on your end. Thank you!

**Comment by gbaned** [comment]
Hi @LukeBoyer Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @LukeBoyer Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @LukeBoyer Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @LukeBoyer Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @LukeBoyer Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @LukeBoyer Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @LukeBoyer Can you please review this PR ? Thank you!

**Comment by keerthanakadiri** [comment]
Hi @LukeBoyer Can you please review this PR ? Thank you!

**Comment by keerthanakadiri** [comment]
Hi @LukeBoyer Can you please review this PR ? Thank you!

**Comment by keerthanakadiri** [comment]
Hi @LukeBoyer Can you please review this PR ? Thank you!

**Comment by keerthanakadiri** [comment]
Hi @LukeBoyer Can you please review this PR ? Thank you!

**Comment by keerthanakadiri** [comment]
Hi @LukeBoyer , Can you please review this PR ? Thank you!

**Comment by keerthanakadiri** [comment]
Hi @LukeBoyer , Can you please review this PR ? Thank you!

**Comment by keerthanakadiri** [comment]
Hi @LukeBoyer, Can you please review this PR ? Thank you!

**Review by LukeBoyer — APPROVED** [review]

**Pull request merged**